src/test/proposals/CoolerV2Proposal.t.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import {ProposalTest} from "./ProposalTest.sol";
+import {Kernel, Actions, Policy, toKeycode} from "src/Kernel.sol";
+import {console2} from "forge-std/console2.sol";
+
+// CoolerV2Proposal imports
+import {CoolerV2Proposal} from "src/proposals/CoolerV2Proposal.sol";
+
+contract CoolerV2ProposalTest is ProposalTest {
+    Kernel public kernel;
+
+    uint48 public constant MS_INSTALLATION_BLOCK = 22490036;
```

Comment:
I think this is a mistake - it should be: `22490361` ?

https://etherscan.io/tx/0x0784c9ac36839e80ebbb5f2ac9079659a860aa36543cbed0bfd6f2b422d5aa89

Reply:
No, there was an MS batch (which this refers to) for installation in the kernel. OCG proposal submission has a flag on line 25 that is set (and causes it to compare calldata).

Reply:
gotcha - that's an uncanny coincidence on the block numbers then `22490361` vs `22490036`